UnitTests/BraintreeThreeDSecureTests/BTThreeDSecureRequest_Tests.swift

```diff
@@ -28,6 +28,13 @@ class BTThreeDSecureRequest_Tests: XCTestCase {
         XCTAssertEqual(request.accountTypeAsString, nil)
     }
 
+    // MARK: - versionRequested
+
+    func testVersionRequested_defaultsToVersion2() {
```

Comment:
should we add a test to prove merchants can set v1 if they want to?

Reply:
We have a [test](https://github.com/braintree/braintree_ios/blob/dad8fe6d02ab41d6d547f40b1a1809b357efd569/UnitTests/BraintreeThreeDSecureTests/BTThreeDSecureRequest_Tests.swift#L56-L99) that if you set `v1` that the appropriate 3DS v1 methods get called. 

We think this suffices in testing that setter.